src/openrct2/CmdlineSprite.cpp

```diff
@@ -77,12 +78,14 @@ void SpriteFile::MakeEntriesAbsolute()
 {
     for (auto& entry : Entries)
         entry.offset += reinterpret_cast<uintptr_t>(Data.data());
+    isAbsolute = true;
 }
 
 void SpriteFile::MakeEntriesRelative()
 {
     for (auto& entry : Entries)
         entry.offset -= reinterpret_cast<uintptr_t>(Data.data());
+    isAbsolute = false;
```

Comment:
(On commit https://github.com/OpenRCT2/OpenRCT2/pull/13590/commits/48b3fe8944c9ece0036c71c037941a7fbc2ada17)

We should probably only do each of these if `isAbsolute` is set to a valid value, e.g., only relativize if absolute and only make absolute if relative. Otherwise we might have some problems.